help/md/fnGammaASRV.md

```diff
@@ -0,0 +1,50 @@
+## name
+fnGammaASRV
+## title
+fnGammaASRV
+## description
+Add Gamma-distributed across-site rate variation (ASRV) to a submodel.
+
+## details
+Each site evolves according to the specified submodel, but at an unknown rate
```

Comment:
I guess so.  Sure.

FWIW, you can think of the Gamma site model as a model that takes a real number (alpha). an integer (n), and a model (the site model) as parameters.  When a model is parameterized by another model, I've been calling that a sub-model.  Do you any suggestions for something else to call it?

I do realize that this is kind of a conceptual jump (models parameterized by other models) so I think its fine to just say "site model" here.

Reply:
It's not a super strong feeling - because I agree submodel makes sense in the context you've described. But it also gets used in hierarchical modeling, where you might have different processes describing different data sources. So if you want to leave it, I think it's fine, but for the precision of the example, site model might be better.

Reply:
OK, staying with "site model" and "site model parameter".